Templates/CSharp/EdgeDriverTemplate/ProjectTemplates/CSharp/Test/EdgeDriverTemplate.vstemplate

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
+  <TemplateData>
+    <TemplateID>Microsoft.CSharp.UnitTestProject.EdgeDriverTest</TemplateID>
+    <Name>Web Driver Test for Edge (.NET Framework)</Name>
+    <Description>Create an automatic UI test using Edge Driver.</Description>
```

Comment:
`A project that contains unit tests that can automate UI testing of web sites within Edge browser.` may read better.

Reply:
Fixed.